pkg/logbridge/logbridge.go

```diff
@@ -0,0 +1,88 @@
+package logbridge
+
+import (
+	"bufio"
+	"io"
+
+	"github.com/square/p2/pkg/logging"
+)
+
+// This type is agnostic of pipes. os.File conforms to io.Reader and io.Writer
+// so it should not be difficult to migrate
+type Bridge struct {
+	src  io.Reader
+	dest io.Writer
+
+	lines         chan []byte
+	needsTeardown bool
+}
+
+// New takes two files src and dest and returns a composite object of the
```

Comment:
It would be useful to make it explicit that `New` is a constructor for a `Bridge`.  What kinds of properties do `src` and `dest` need to provide?  Why must they be files?  How are the files composed?